quarkus-cli/src/test/java/io/quarkus/ts/quarkus/cli/QuarkusCliExtensionsIT.java

```diff
@@ -29,6 +31,7 @@
 @QuarkusScenario
 @DisabledOnQuarkusVersion(version = "1\\..*", reason = "Quarkus CLI has been reworked in 2.x")
 @DisabledIfSystemProperty(named = "profile.id", matches = "native", disabledReason = "Only for JVM verification")
+@EnabledOnQuarkusVersion(version = ".*redhat.*", reason = "There is no stream 2.7 in code.quarkus.io anymore")
```

Comment:
GH CI is running upstream quarkus version, and we should keep this behavior as it is.

Reply:
Well, I proceed with this amend

Reply:
You can't run 2.7 against upstream as it won't work. Let's try it out.